.github/workflows/self-push-amd.yml

```diff
@@ -244,6 +251,106 @@ jobs:
           name: ${{ matrix.machine_type }}_run_all_tests_gpu_${{ env.matrix_folders }}_test_reports
           path: /transformers/reports/${{ matrix.machine_type }}_tests_gpu_${{ matrix.folders }}
 
+  run_tests_multi_gpu:
+    name: Model tests
+    needs: setup_gpu
+    # `dummy` means there is no test to run
+    if: contains(fromJson(needs.setup.outputs.matrix), 'dummy') != true
+    strategy:
+      fail-fast: false
+      matrix:
+        folders: ${{ fromJson(needs.setup.outputs.matrix) }}
+        machine_type: [ multi-gpu ]
```

Comment:
I think we can have a single job `run_tests_gpu` but with `machine_type: [single-gpu,multi-gpu ]`.

For current CI, we separate this as they have different docker options: `--gpus 0` and `--gpus all`.

But for this AMD workflow, they are the same.

Reply:
Done ✅